go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/sasha-s/go-deadlock v0.3.1
 	github.com/snikch/goodman v0.0.0-20171125024755-10e37e294daa
 	github.com/spf13/cobra v1.8.0
-	github.com/spf13/viper v1.18.1
+	github.com/spf13/viper v1.15.0
```

Comment:
sorry I'm late but this may get reverted by dependabot so we could add a replace statement to stay pinned to this version